drafts/SC1-2-2-captions-prerecorded.md

```diff
@@ -0,0 +1,171 @@
+
+## Test properties
+| Property                  | Value
+|---------------------------|------------------------------------------|
+| Test name                 | Captions on prerecorded synchronized media
+| Test requirement          | SC1-2-2-captions-prerecorded
+| Test mode                 | manual
+| Test environment          | rendered page + server connection
+| Test subject              | single web page
+| User expertise and skills | no prior knowledge
+| User profile              | Requires vision + hearing
+## Test procedure
+
+### Selector
+Test method: [manual]
+
+All pages including elements capable of playing video.
+
+Those can be identified by including elements having an attribute value containing a media file format, such as  .mov, .rm, .ram, .asf, .swf, .dcr, .avi, .mpg, .mpeg, .mp4, .m4v, .webm, .flv, .isma
+
+    XPath: //*[contains(@*,'.mov') or contains(@*,'.rm') or ... ]
+
+### Step 1
+Test method: [manual]
+
+Check that the content is a video containing an audio-track.
+
+**User Input Question:**
+
+| Property             | Value
+|----------------------|-------
+| presented-item       | Media content
+| requires-context     | yes
+| requires-interaction | yes
+| question             | Does the media consist of video combined with audio?
+| help                 | Please check that the audio is not muted. If there is audio content in the media, select “Yes”. Else select “No”.
+
+If yes, continue with [Step 2](#Step 2)
+
+else return:
+
+| Outcome  | Inapplicabe
+|----------|-----
+| Testcase | SC1-2-2-captions-prerecorded
+| ID       | SC1-2-2-captions-prerecorded-inapplicable1
+
+
+### Step 2
+Test method: [manual]
+
+Check that the video-only content is prerecorded.
+
+**User Input Question:**
+
+| Property             | Value
+|----------------------|-------
+| presented-item       | Whole page
+| requires-context     | yes
+| requires-interaction | yes
+| question             | Is the video-only content prerecorded?
+| help                 | Usually live content is explicitly marked as such. You can also try to navigate to the end of the media. On live content you will not be able to. If the content is prerecorded, select “Yes”. Else select “No”.
+
+
+If yes, continue with [Step 3](#Step 3)
+
+else return:
+
+| Outcome  | Inapplicable
+|----------|-----
+| Testcase | SC1-2-2-captions-prerecorded
+| ID       | SC1-2-2-captions-prerecorded-inapplicable2
+
+
+## Step 3
+Test method: [manual]
+
+Check that the prerecorded video content has relevant information for the context of the web page.
+
+**User Input Question:**
+
+| Property             | Value
+|----------------------|-------
+| presented-item       | Whole page
+| requires-context     | yes
+| requires-interaction | yes
+| question             | Does the video content not contain information relevant for the context?
```

Comment:
Alister: Maybe talk something about "is the video meaningful in the context of the page".